worksheet_b/README.MD

```diff
@@ -0,0 +1,53 @@
+
+
+Print Welcome to
+Terminal 4
+
+Print Password
+Required
+
+Secret Word = Water
+
+Player Strikes Left
+= 5
+
+Computer generates
+the 11 words in a random order
+
+                Computer words = ("Bales,
+Towel, Water, Artsy, Fairy, Chair, Wacky, Voice, Dough,            Grass, Knee, Water.")
+
+If Player word is
```

Comment:
This seems to be out of order -- the player word is being checked before "player guesses a word"